From d493b65daeb4a63ac356d4b2adffaba475c1bfdf Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Wed, 28 Jul 2021 18:28:22 +0300
Subject: [PATCH] [SW-45031] gaudi2: add dummy HL_TARGET files

Added HL_TARGET specific (asic, onix, pldm) dummy
files. These will be used to move HL_TARGET
specific common files for GAUDI2.

Change-Id: Ib2335906b3a58f11cf872483b10f1dd9d816a9b7
Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
---

diff --git a/arch/gaudi2/common/Makefile b/arch/gaudi2/common/Makefile
index 8b002b4..e211ca5 100644
--- a/arch/gaudi2/common/Makefile
+++ b/arch/gaudi2/common/Makefile
@@ -1,3 +1,5 @@
 PREBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH)/../common/common.c)
+PREBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH)/../common/$(HL_TARGET).c)
 
 PPBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH)/../common/common.c)
+PPBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH)/../common/$(HL_TARGET).c)
diff --git a/arch/gaudi2/common/asic.c b/arch/gaudi2/common/asic.c
new file mode 100644
index 0000000..f593276
--- /dev/null
+++ b/arch/gaudi2/common/asic.c
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier:     GPL-2.0+
+/*
+ *
+ * (C) Copyright 2021 HabanaLabs, Ltd.
+ *
+ */
+
+/* TODO: Gaudi2 asic(HL_TARGET) specific common functions to be moved here */
diff --git a/arch/gaudi2/common/onix.c b/arch/gaudi2/common/onix.c
new file mode 100644
index 0000000..e76553e
--- /dev/null
+++ b/arch/gaudi2/common/onix.c
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier:     GPL-2.0+
+/*
+ *
+ * (C) Copyright 2021 HabanaLabs, Ltd.
+ *
+ */
+
+/* TODO: Gaudi2 onix(HL_TARGET) specific common functions to be moved here */
diff --git a/arch/gaudi2/common/pldm.c b/arch/gaudi2/common/pldm.c
new file mode 100644
index 0000000..f5f8caf
--- /dev/null
+++ b/arch/gaudi2/common/pldm.c
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier:     GPL-2.0+
+/*
+ *
+ * (C) Copyright 2021 HabanaLabs, Ltd.
+ *
+ */
+
+/* TODO: Gaudi2 pldm(HL_TARGET) specific common functions to be moved here */
